python/pyarrow/_dataset.pyx

```diff
@@ -92,6 +92,20 @@ cdef class Dataset:
         else:
             return Expression.wrap(expr)
 
+    def GetFragments(self, columns=None, filter=None):
```

Comment:
```suggestion
    def get_fragments(self, columns=None, filter=None):
```

PEP8 in python -> only classes use CamelCase

(and same for the other occurences)

Reply:
alright